packages/travelmakers-design-core/src/components/Tab/PrimaryTab/stories/PrimaryTab.stories.tsx

```diff
@@ -0,0 +1,41 @@
+import { Meta } from "@storybook/react";
+import { useState } from "react";
+import { PrimaryTab } from "../PrimaryTab";
+
+const data = ["서울", "제주", "경상", "수도권"];
+
+export default {
+  title: "@travelmakers-design-v2/core/General/Tab/PrimaryTab/PrimaryTab",
+  component: PrimaryTab,
+  argTypes: {
+    items: {
+      control: { type: "array" },
+      description: "Tab에 표시될 리스트 입니다. PrimaryTab.Item를 사용합니다.",
+      table: {
+        type: {
+          summary: "React.ReactNode[]",
+        },
+      },
+    },
+  },
+} as Meta;
+
+export const Default = () => {
```

Comment:
p3; 스토리부분에 default value등은 기본적으로 넣어주는게 좋지 않을까요?
`show code`로 보게될 경우 빈칸으로만 표시되다보니 가독성이 떨어질 수 있을 것 같습니다.

```suggestion
export const Default = (props) => {
```
형태로 하게되면 위에서 작성한 default value를 가져올 수 있습니다~!

Reply:
default value는 실제 해당 prop의 default value가 들어가야하지 않을까요? 아니면 지금처럼 스토리북의 예시 데이터라를 넣어주는게 맞을까요?

Reply:
실제 props라기보다는 스토리북의 예시 데이터로써 default value를 사용한다고 알고 있습니다. 넣어주는게 좀 더 깔끔하지 않을까 저는 생각하는데 어떻게 생각하시나요?

Reply:
일단 스토리북 7.0부터는 default value는 deprecated되었다고 하니 사용하지 않는게 좋을 것 같습니다!

Reply:
https://github.com/storybookjs/storybook/blob/next/MIGRATION.md#no-longer-inferring-default-values-of-args